src/wifes_wsol.py

```diff
@@ -1158,7 +1158,7 @@ def derive_wifes_polynomial_wave_solution(inimg,
         else:
             grating = arc_hdr['GRATINGB']
     if arc_name == None:
-        init_arc_name = arc_hdr['M1ARCLMP']

```

Comment:
1 place in pywifes_utils.py that uses this header still. As in the comment above, maybe worth fixing? But I don't know if it's used or defunct.

Reply:
I have 'LAMP'... I think you didn't spot the new changes in the line highlighted in green. Can you please double-check?

Reply:
This line is fine, I meant in `pywifes_utils.py`, similar to this comment: https://github.com/PyWiFeS/pipeline/pull/25#discussion_r1365306651

I Think that these utils functions are meant to be called by users directly. That'll make it tricky to design test cases for? But we can come back to it later.

Reply:
Good point. I'll make sure to have test cases for all the scripts we'll need to maintain so we can check all these issues in all of them. For now, as I said before, let's focus just on the pipeline scripts.